detox/local-cli/utils/patchJestUtil.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const path = require('path');
+const resolveFrom = require('resolve-from');
+const log = require('../../src/utils/logger').child({ cat: 'jest-patch' });
+
+function patchJestUtil() {
+  try {
+    const jestUtilPath = resolveFrom(process.cwd(), 'jest-util/package.json');
+    const isInteractivePath = path.join(path.dirname(jestUtilPath), 'build/isInteractive.js');
+
+    if (!fs.existsSync(isInteractivePath)) {
+      log.warn('Could not find node_modules/jest-util/build/isInteractive.js to patch!');
+      return;
+    }
+
+    const content = fs.readFileSync(isInteractivePath, 'utf8');
+    if (!content.includes('DETOX_REPL')) {
+      const patchedContent = content.replace(
+        "process.env.TERM !== 'dumb'",
```

Comment:
perhaps too tightly coupled with the implementation?
What about `exports.default = _default;` ==> `exports.default = _default && !DETOX_REPL;` or something like that

Reply:
Only if you really really insist... 🤷‍♂️